react/features/speaker-stats/middleware.js

```diff
@@ -35,8 +39,13 @@ MiddlewareRegistry.register(({ dispatch, getState }) => next => action => {
             const stats = filterBySearchCriteria(state, speakerStats);
             const pendingReorder = getPendingReorder(state);
 
-            dispatch(updateStats(pendingReorder ? getSortedSpeakerStats(state, stats) : stats));
+            if (pendingReorder) {
+                dispatch(updateSortedSpeakerStatsIds(getSortedSpeakerStatsIds(state, stats)));
```

Comment:
Now when `pendingReorder` is true we dispatch an action to update the sorted speakers